app/models/shape_point.rb

```diff
@@ -0,0 +1,4 @@
+class ShapePoint < ActiveRecord::Base
+  belongs_to :shape
+  has_one :agency, through: :shape
```

Comment:
Do we really need the ShapePoint.agency? has_one :through doesn't give us anything at the data layer, so I'm just wondering if we use that anywhere?